fountain-retrofit/src/main/java/com/xmartlabs/fountain/retrofit/adapter/RetrofitNetworkDataSourceAdapterFactory.kt

```diff
@@ -63,26 +63,22 @@ object RetrofitNetworkDataSourceAdapterFactory {
  * Provides a [RetrofitNetworkDataSourceAdapter] implementation of a [ListResponseWithEntityCount] response.
  * It is used when the service returns the entity count in the response.
  *
- * @param Value The value that the service returns.
- * @param ListResponseValue The response type that the service returns.
- * @param pageFetcher It is used to fetch each page from the service.
+ * @param ServiceResponse The response type that the service returns.
  * @param firstPage The first page number, defined by the service.
  */
-fun <Value, ServiceResponse : ListResponseWithEntityCount<Value>>
-    RetrofitPageFetcher<ServiceResponse>.toTotalEntityCountRetrofitNetworkDataSourceAdapter(
-    firstPage: Int = com.xmartlabs.fountain.common.FountainConstants.DEFAULT_FIRST_PAGE
+fun <ServiceResponse : ListResponseWithEntityCount<*>>
+    RetrofitPageFetcher<ServiceResponse>.toTotalEntityCountNetworkDataSourceAdapter(
+    firstPage: Int = FountainConstants.DEFAULT_FIRST_PAGE
 ) = RetrofitNetworkDataSourceAdapterFactory.fromTotalEntityCountListResponse(this, firstPage)
 
 /**
  * Provides a [RetrofitNetworkDataSourceAdapter] implementation of a [ListResponseWithEntityCount] response.
  * It is used when the service returns the page count in the response.
  *
- * @param Value The value that the service returns.
- * @param ListResponseValue The response type that the service returns.
- * @param pageFetcher It is used to fetch each page from the service.
+ * @param ServiceResponse The response type that the service returns.
```

Comment:
maybe we can change all these for something like `@param ServiceResponse The response type returned by the service.`? What do you think?